packages/shared/src/components/cards/WelcomePostCard.tsx

```diff
@@ -26,6 +26,7 @@ export const WelcomePostCard = forwardRef(function SharePostCard(
     children,
     enableSourceHeader = false,
     domProps = {},
+    onDownvoteClick,
```

Comment:
I really wish we had one source to only mod this (no action needed maybe good backlog)